### DISCUSSION TITLE
Assistance Needed: How to switch the project to .NET and MAUI 9.0

### DISCUSSION THREAD

**Discussion by GraceDong** [CommunityToolkit/Maui#2155] [Q&A] [answered]
We want to test CommunityToolkit.Maui.Sample.sln with .NET and MAUI 9.0. But we hit some problems with the dependency package. Could someone help me take a look?

**Steps:**

1. Install latest preview VS on my Windows machine
2. Git clone the project https://github.com/CommunityToolkit/Maui.git
3. Change the SDK version to 9.0 in global,json as follows:
```
    {
      "sdk": {
       "version": "9.0.100-preview.7.24407.12",
       "rollForward": "latestFeature",
       "allowPrerelease": false
    }
    }
```
4. Change the <NETVersion> and <MauiPackageVersion> to 9.0 in Directory.Build.props as follows:
```
    <NetVersion>net9.0</NetVersion>
    <MauiPackageVersion>9.0.0-preview.7.24408.5</MauiPackageVersion>
```
5. Open CommunityToolkit.Maui.Sample.sln in VS
6. Check the Dependencies are upgraded to net9.0-x
7. Go to Error List window, I saw the below restore package errors:
![image](https://github.com/user-attachments/assets/b1b7cff5-f1f8-4f8d-9644-cc8f58417c19)

I don’t know which packages need updating. I would appreciate any assistance!

**Comment by TheCodeTraveler** [comment, marked as answer]
We don't yet support .NET 9.

We will add .NET 9 support when it GAs in November.
